homeassistant/components/github/__init__.py

```diff
@@ -38,14 +38,15 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         )
 
         await coordinator.async_config_entry_first_refresh()
+        await coordinator.subscribe()
+        hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, coordinator.unsubscribe)
```

Comment:
We don't need this since we do it in `subscribe`.
```suggestion
```

Reply:
Oh, nice! forgot I had it in there as well :smile: